File: Enalotto.vb
Module Enalotto

    Sub Main()
        Dim numbers As SortedSet(Of Int16) = New SortedSet(Of Int16)
        Dim rnd As New Random()

        While numbers.Count < 6
            numbers.Add(rnd.Next(1, 91))
        End While

        Dim index As Integer = 0
        For Each number As String In numbers
            If (number < 10) Then
                Console.Write("0")
            End If

            Console.Write(number)

            If (index < 5) Then
                Console.Write(" - ")
            End If

            index += 1
        Next
    End Sub

End Module

File: Enalotto.cs
using System;
using System.Collections.Generic;

namespace com.prontobenedict.enalotto
{
    class Enalotto
    {
        static void Main(string[] args)
        {
            SortedSet<int> numbers = new SortedSet<int>();
            Random rnd = new Random();

            while(numbers.Count < 6) {
                numbers.Add(rnd.Next(1, 91));
            }

            int index = 0;
            foreach(int number in numbers) {
                if(number < 10) Console.Write("0");
                Console.Write(number);
                if(index < 5) Console.Write(" - ");
                index++;
            }
        }
    }
}
